.hadolint.yaml

```diff
@@ -1,8 +1,11 @@
+---
+
```

Comment:
Not a fan of it, but can live with it

Reply:
This was discussed when yamllint was introduced, I managed to convince reviewers it's a good habit. Unlikely we would need it to actually separate any YAML documents, but a good habit.

Reply:
@LecrisUT anything left to discuss here? If not, please resolve the conversation.

Reply:
I cannot resolve them, but you can mark both of them as resolved